extensions/wikia/FounderEmails/events/FounderEmailsDaysPassedEvent.class.php

```diff
@@ -79,13 +79,13 @@ private function getEmailController( $activateDay ) {
 	}
 
 	public static function register( $wikiParams, $debugMode = false ) {
-		global $wgFounderEmailsEvents;
+		global $wgFounderEmailsEvents, $wgCityId;
 
 		$founderEmailObj = FounderEmails::getInstance();
 
 		$wikiFounder = $founderEmailObj->getWikiFounder();
-		$wikiFounder->setLocalPreference( "founderemails-joins", true );
-		$wikiFounder->setLocalPreference( "founderemails-edits", true );
+		$wikiFounder->setLocalPreference( "founderemails-joins", true, $wgCityId );
+		$wikiFounder->setLocalPreference( "founderemails-edits", true, $wgCityId );
```

Comment:
In that case, should it be an error if you try and set a local preference and it's null at the Service layer in PHP?